ocr_service/main.py

```
@@ -0,0 +1,55 @@
+import os
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+)
+logger = logging.getLogger(__name__)
+
+# Debug: Print environment variables to help diagnose issues
+logger.info("==== DEBUG: Environment Variables ====")
+logger.info(f"GEMINI_API_KEY exists: {'Yes' if 'GEMINI_API_KEY' in os.environ else 'No'}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)